app/services/streaming/streaming.ts

```diff
@@ -707,7 +805,23 @@ export class StreamingService
     }
 
     if (this.state.streamingStatus === EStreamingState.Ending) {
-      obs.NodeObs.OBS_service_stopStreaming(true);
+      if (this.views.isGreen) {
+        if (this.views.enabledPlatforms.length > 1) {
+          this.views.contextsToStream.forEach(async (contextName: string) => {
```

Comment:
shouldnt this be the same regardless of if `enabledPlatforms` is one or more? seems like a bit of a superfluous if step here (and also in the previous instance of this)

Reply:
This is to guarantee that the Green code is skipped for now since it will always return false. The logic inside this if statement what will actually handle the Green functionality. I can also just remove this block.